### ISSUE TITLE
Configuration value of the repositories / Service naming

### ISSUE THREAD

**Issue by lunetics** [prooph/event-store-symfony-bundle#5]
I propose to use a different naming for the repositories.

Given:
```yaml
prooph_event_store:
    stores:
        foo_store:
            type: 'Prooph\EventStore\InMemoryEventStore'
            repositories:
                foo_collection:
                    repository_class: 'My\Own\Repository'
                    aggregate_type: 'My\Aggregate\Type'
```
will create a service named '@foo_collection', which is prone to errors (no namespace etc.). Upcoming symfony versions propose or use alread the FQCN as service-id, which makes it unique.

So a new configuration could look like:

```yaml
prooph_event_store:
    stores:
        foo_store:
            type: 'Prooph\EventStore\InMemoryEventStore'
            repositories:
                My\Own\Repository:                
                    aggregate_type: 'My\Aggregate\Type'
```
There we enforce the FQCN for once as service-id, which is at the same time the repositoryclass.

I'm not sure if we should give it as "hint" in the docs, or enforcing it by omitting the 'repository_class' key, enforcing the use of FQCN.

What are your thoughts?

**Comment by codeliner** [comment]
I would not force it, but suggest it in the docs. F.e. I prefer to have a `UserCollection` or `UserRepository` interface and use that as service id instead of the concrete implementation.

**Comment by UFOMelkor** [comment]
AFAIK are FQCN as service ids not yet widely used. Therefore forcing it would IMHO be hard and an unnecessary hurdle.
But I have nothing against making the `repository_class` optional. Less necessary configuration is (mostly) good :-)

**Comment by UFOMelkor** [comment]
Implemented in #32
